README.md

```diff
@@ -151,5 +151,5 @@ Snapshots are uploaded to [Sonatype](https://oss.sonatype.org/content/repositori
 commits to master.
 
 ### Docker Images
-Released versions of zipkin-server are published to Docker Hub as `openzipkincontrib/zipkin-storage-kafka` and GitHub
```

Comment:
note: this at least temporarily unsupports Docker registry push. Adding it back later is possible albeit confusing.